content/docs/reference-react.md

```diff
@@ -60,20 +60,20 @@ See [Using React without JSX](/docs/react-without-jsx.html) for more information
 
 ### Suspense {#suspense}
 
-Suspense lets components "wait" for something before rendering. Today, Suspense only supports one use case: [loading components dynamically with `React.lazy`](/docs/code-splitting.html#reactlazy). In the future, it will support other use cases like data fetching.
+*Suspense* permite que componentes "esperem" por algo antes de renderizar. Atualmente, *Suspense* suporta somente somente um caso de uso: [carregando componentes dinamicamente com `React.lazy`](/docs/code-splitting.html#reactlazy). Futuramente, ele suportará outros casos de uso, como busca de dados.
 
 - [`React.lazy`](#reactlazy)
 - [`React.Suspense`](#reactsuspense)
 
 ### Hooks {#hooks}
 
-*Hooks* are a new addition in React 16.8. They let you use state and other React features without writing a class. Hooks have a [dedicated docs section](/docs/hooks-intro.html) and a separate API reference:
+*Hooks* são uma novidade no React 16.8. Eles permitem que você utilize estado e outras funcionalidades do React sem ter que escrever uma classe para isso. *Hooks* possuem uma [seção dedicada na documentação](/docs/hooks-intro.html) e uma referência da *API* separada:
```

Comment:
```suggestion
Os *Hooks* são uma novidade no React 16.8. Eles permitem que você utilize o estado (_state_) e outras funcionalidades do React sem ter que escrever uma classe para isso. *Hooks* possuem uma [seção dedicada na documentação](/docs/hooks-intro.html) e uma referência da *API* separada:
```

Reply:
Here, I'd only add "Os" before "Hooks" on the second second phrase:

> Os *Hooks* são uma novidade no React 16.8. Eles permitem que você utilize o estado (*state*) e outras funcionalidades do React sem ter que escrever uma classe para isso. **Os** *Hooks* possuem uma [seção dedicada na documentação](/docs/hooks-intro.html) e uma referência da *API* separada:

Reply:
sounds good 👍